### PR TITLE
fix chatbot example

### PR DESCRIPTION
**Pull request by feipenghe**
This PR fixes https://github.com/asyml/forte/issues/700. 

### Description of changes
* add required packages in `README.md`
* ~~add `__init__.py` in `ftx` for importing~~
* corrected some updated configuration

### Possible influences of this PR.
After the change, the chatbot example should be able to run without configuration error.

### Test Conducted
Describe what test cases are included for the PR.

### CODE REVIEW

File: examples/chatbot/README.md [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
+After setting up `forte` environment by installing packages in `requirements.txt` in the project root folder.
+```
+pip install termcolor
+pip install tensorflow==2.4 # version compantible with numpy==1.19.5
+pip install faiss-gpu
+```
+
+Then we need to install forte external pacakges
+```
+pip install src/faiss
```
Comment (hunterhector) [inline examples/chatbot/README.md:26]: This instruction is not clear, since it didn't specify the repo you need to download and the directory you are in.

**Review by hunterhector — COMMENTED** [review]
We deliberately remove `__init__.py` from `ftx`, and because we use find_namespace_package `https://github.com/asyml/forte/blob/master/setup.py#L20`. It should work with `pip install` even if the file does not present.

It is important that we don't have this file to avoid conflicting with our other projects.

File: examples/chatbot/README.md (hunk 1 above)
Comment (feipenghe) [reply examples/chatbot/README.md:27]: changed the description.

**Comment by bhaskar2443053** [comment]
HI @feipenghe ,
the chatbot example still throws this error, while running it.

image

**Pull request merged**